packages/storefront-lwc/src/modules/commerce/productPrice/productPrice.scss

```diff
@@ -25,3 +25,13 @@
   color: $light-gray;
   font-weight: $font-weight-normal;
 }
+.basket {
```

Comment:
again, will this css work for when this component is used in a different place where the parent does not have the css class?

Reply:
If this component is used somewhere else, they will have to provide a different context and give its own CSS. This context is used because the LWC does not allow parents reach into child for styling. This is being use for productPrice component.

Reply:
ok ok thanks for explaining, makes sense.